tools/migrate/main.go

```diff
@@ -255,6 +268,9 @@ func migrate(srcDir, dstDir string) error {
 	}
 
 	snapshot := v1manifest.NewSnapshot(initTime)
+	if expires != 0 {
```

Comment:
Do we need to set different expiration for different roles?

Reply:
only used to set as a very high value for the test.
if not settled, it will used different value in New**